src/python/pants/backend/python/rules/download_pex_bin.py

```diff
@@ -28,7 +28,7 @@ class PexBin(ExternalTool):
     default_version = "v2.1.13"
     default_known_versions = [
         f"v2.1.13|{plat}|240712c75bb7c7cdfe3dd808ad6e6f186182e6aea3efeea5760683bb0fe89198|2633838"
-        for plat in ["darwin", "linux"]
+        for plat in ["darwin", "linux "]
```

Comment:
This seems like a regression.

Reply:
No, this is deliberate, so the lines align nicely.